src/Agent.Listener/Configuration/ConfigurationManager.cs

```diff
@@ -373,7 +364,22 @@ public AgentSettings LoadSettings()
                     throw new Exception(StringUtil.Loc("UnconfigureOSXService"));
 #endif
                 }
-
+                else
+                {
+#if OS_WINDOWS
+                    //running as process, unconfigure autologon if it was configured                    
+                    if (_store.IsAutoLogonConfigured())
+                    {
+                        var autoLogonConfigManager = HostContext.GetService<IAutoLogonManager>();
+                        autoLogonConfigManager.Unconfigure();                        
+                    }
+                    else
+                    {
+                        Trace.Info("AutoLogon was not configured on the agent.");
+                    }
```

Comment:
for tracing only.

---
In reply to: [119684715](https://github.com/Microsoft/vsts-agent/pull/985#discussion_r119684715) [](ancestors = 119684715)